backend/src/api/query.rs

```diff
@@ -96,9 +96,12 @@ impl Query {
         search::perform(&query, context).await
     }
 
-    /// Searches through all events that the user has write access to
-    /// (unlisted and listed). If `writeable_only` is `false`, events that are
-    /// listed and that the user has read access to are also returned.
+    /// Searches through events. Searches through:
```

Comment:
Maybe write sth like
```suggestion
    /// Searches through events. Results include:
```
Same in line 113. Sounds a little more natural to me, though your version is also fine.